<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Netsim_MainWindow</class>
 <widget class="QMainWindow" name="Netsim_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Netsim_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>176</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnOnOff">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="widgetConfig" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>151</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>143</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutConfig">
      <item>
       <widget class="QGroupBox" name="gpDatasource">
        <property name="title">
         <string>Datasource</string>
        </property>
        <widget class="QRadioButton" name="radioDataRand">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Rando&amp;m</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioDataInput">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>User Input</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Data Length</string>
        </property>
        <widget class="QSpinBox" name="spinDataLength">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>23</y>
           <width>111</width>
           <height>29</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Window size</string>
        </property>
        <widget class="QSpinBox" name="spinDataLength_2">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>24</y>
           <width>111</width>
           <height>29</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>ARQ Type</string>
        </property>
        <widget class="QRadioButton" name="radioArqSw">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop-Wait</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioArqBn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Bac&amp;k N</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Simulation Speed</string>
        </property>
        <widget class="QSlider" name="slidSimSpeed">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Send Status</string>
    </property>
    <widget class="StatusWidget" name="widghtStatus" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Operation">
    <property name="title">
     <string>Sta&amp;rt/Stop</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>&amp;Quit</string>
    </property>
   </widget>
   <addaction name="menu_Operation"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
   <addaction name="menuQuit"/>
  </widget>
  <action name="actionStart_Stop">
   <property name="text">
    <string>Start/Stop</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusWidget</class>
   <extends>QWidget</extends>
   <header>statuswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
